website/addons/mendeley/tests/test_models.py

```diff
@@ -43,406 +33,22 @@ def test_handle_callback(self, mock_get_client):
-        assert_true(
-            self.user_settings.verify_oauth_access(
-                node=self.node,
-                external_account=external_account,
-                metadata={'folder': 'fake-folder-id'}
-            )
-        )
-
-        log = self.node.logs[-1]
-        assert_equal(log.action, 'mendeley_folder_selected')
-        assert_equal(log.params['folder_id'], folder_id)
-        assert_equal(log.params['folder_name'], folder_name)
-
-    def test_has_auth_false(self):
-        external_account = MendeleyAccountFactory()
-
-        assert_false(self.node_settings.has_auth)
-
-        # both external_account and user_settings must be set to have auth
-        self.node_settings.external_account = external_account
-        assert_false(self.node_settings.has_auth)
-
-        self.node_settings.external_account = None
-        self.node_settings.user_settings = self.user_settings
-        assert_false(self.node_settings.has_auth)
-
-        # set_auth must be called to have auth
-        self.node_settings.external_account = external_account
-        self.node_settings.user_settings = self.user_settings
-        assert_false(self.node_settings.has_auth)
-
-    def test_has_auth_true(self):
-        external_account = MendeleyAccountFactory()
-        self.user.external_accounts.append(external_account)
-
-        self.node_settings.set_auth(external_account, self.user)
-
-        # mendeley_list_id should have no effect
-        self.node_settings.mendeley_list_id = None
-        assert_true(self.node_settings.has_auth)
-
-        # mendeley_list_id should have no effect
-        self.node_settings.mendeley_list_id = 'totally fake ID'
-        assert_true(self.node_settings.has_auth)
-
-    def test_selected_folder_name_root(self):
-        self.node_settings.mendeley_list_id = 'ROOT'
-
-        assert_equal(
-            self.node_settings.selected_folder_name,
-            "All Documents"
-        )
-
-    def test_selected_folder_name_empty(self):
-        self.node_settings.mendeley_list_id = None
-
-        assert_equal(
-            self.node_settings.selected_folder_name,
-            ''
-        )
-
-    @mock.patch('website.addons.mendeley.model.Mendeley._folder_metadata')
-    def test_selected_folder_name(self, mock_folder_metadata):
-        # Mock the return from api call to get the folder's name
-        mock_folder = mock.Mock()
-        mock_folder.name = 'Fake Folder'
-
-        # Add the mocked return object to the mocked api client
-        mock_folder_metadata.return_value = mock_folder
-
-        self.node_settings.mendeley_list_id = 'fake-list-id'
-
-        assert_equal(
-            self.node_settings.selected_folder_name,
-            'Fake Folder'
-        )
-
-    # TODO: Make these tests generic and move to core
-
-    @mock.patch('framework.status.push_status_message')
-    def test_remove_contributor_authorizer(self, mock_push_status):
```

Comment:
Are these tests not needed anymore? I couldn't find if they were moved to somewhere else.